ts/output/common/Wrapper.ts

```diff
@@ -450,18 +453,29 @@ export class CommonWrapper<
   /**
    * Easy access to the linebreak visitor
    *
-   * @returns The linebreak visitor
+   * @returns {Linebreaks} The linebreak visitor
    */
-  get linebreaks() {
+  /* prettier-ignore */
+  get linebreaks(): Linebreaks<
+    N, T, D,
+    CommonOutputJax<N, T, D, WW, WF, WC, CC, VV, DD, FD, FC>,
+    WW, WF, WC, CC, VV, DD, FD, FC> {
     return this.jax.linebreaks;
   }
 
   /**
    * Easy access to the linebreak options
    *
-   * @returns The linebreak options
-   */
-  get linebreakOptions() {
+   * @returns {{inline: boolean,
+   *   width: string,
+   *   lineleading: number,
+   *   LinebreakVisitor: null}} The linebreak options
+   */
+  get linebreakOptions(): {
+    inline: boolean;
+    width: string;
+    lineleading: number;
```

Comment:
Is `LinebreakVisitor` left out here for a reason?

Reply:
Yes, because its generic type is a bit of a nightmare: https://github.com/mathjax/MathJax-src/blob/aa3c017e317a4ce768bccd6a0385c7f15694bbf1/ts/output/common/LinebreakVisitor.ts#L145
However, it turns out leaving it as `null` actually works, too. So I am adding that.